### DISCUSSION TITLE
SSO OAuth2 not working, Login Error: AADSTS700016

### DISCUSSION THREAD

**Discussion by mukifl** [netbox-community/netbox#9624] [Q&A]
Hello Everybody

I recently setup netbox and now wanted to enable SSO for users to sign in. 
I went through the documentation and followed the guide step by step but im I am getting below error. 

![image](https://user-images.githubusercontent.com/108263878/176381607-be991e26-c033-4748-880f-104bffb5b0cb.png)

Netbox Version: v3.2.4
Python Version: Python 3.9

I already checked threads on the www and the most common case is, that the application ID is wrong. 
I did check the application id many times and copy-pasted the value and its definitely the right one in the configuration.py. 

Please find below my configuration.py config: 

![image](https://user-images.githubusercontent.com/108263878/176382168-1af57767-c58d-41bb-ad0c-bd8750323bf9.png)

Any help or suggestions are very much appreciated. 

Thank you in advance. 

Cheers

**Comment by candlerb** [comment]
The setting names are case-sensitive.  You wrote `..._OAUTH2_Key` but it should be `..._OAUTH2_KEY`

See: https://python-social-auth.readthedocs.io/en/latest/backends/azuread.html
